debian/entry.sh

```diff
@@ -90,6 +90,8 @@ function init_non_systemd()
 		"$CMD" "$@" &
 		pid=$!
 		wait "$pid"
+		exit_code=$?
+		fg > /dev/null || exit "$exit_code"
```

Comment:
this PR doesn't have a "fixes #250" anywhere. Also, the issue doesn't explain why the issue happens, it just states that "interactive shell doesn't work". Can you explain the reason and what is going on with the script, the job, background, foreground etc?